sync-core/src/main/java/com/cloudant/sync/datastore/RevisionHistoryHelper.java

```diff
@@ -160,7 +161,8 @@ private static boolean checkRevisionStart(DocumentRevs.Revisions revisions) {
      */
     public static MultipartAttachmentWriter createMultipartWriter(List<BasicDocumentRevision> history,
                                                                   List<? extends Attachment> attachments,
-                                                                  PushAttachmentsInline inlinePreference) {
+                                                                  PushAttachmentsInline inlinePreference,
+                                                                  int minRevPos) {
```

Comment:
Add argument to javadoc (possibly in other places -- add new doc if need be -- too as it's rather unclear from the var name).